metaflow_extensions/ray/plugins/ray_decorator.py

```
@@ -278,6 +296,11 @@ def setup_distributed_env(self, flow):
         main_port = self._resolve_port()
         main_ip = resolve_main_ip()
         start_ray_processes(
-            self.ubf_context, main_ip, main_port, current.parallel.node_index
+            self.ubf_context,
+            main_ip,
+            main_port,
+            current.parallel.node_index,
+            logging_level=self.attributes["ray_logging_level"],
+            log_style=self.attributes["ray_log_style"],
```

Can we have one simple validation for `log_style` (to fail fast). Ideally this can even go in `step_init` callback

---

Yep, added to make sure the options are valid